poyoweb-server/src/db/db.ts

```diff
@@ -0,0 +1,53 @@
+import 'dotenv/config';
+import { drizzle } from 'drizzle-orm/node-postgres';
+import { eq } from 'drizzle-orm';
+import * as schema from './schema';
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+import { v4 as uuidv4 } from 'uuid';
+const db = drizzle(process.env.DB_URL!, { schema });
+
+export function readDb() {
+  return db.select().from(schema.usersTable);
+}
+
+export async function registerUser(email: string, password: string, name: string) {
+  password = bcrypt.hashSync(password, 10);
+  const answer = await db.insert(schema.usersTable).values({ email, password, name }).returning();
+  return (await createSession(answer[0].id, new Date(Date.now() + 86400000), 'unknown')).jwt_token;
+}
+
+export async function verifyUser(email: string, password: string) {
+  const user = await db.select().from(schema.usersTable).where(eq(schema.usersTable.email, email));
+  if (user.length === 0) {
+    return false;
+  }
```

Comment:
you're returning early if user is not found. this means youre giving the attack the information that the account doesnt exist (called user enumeration, its a security vulnerability)

what attackers can do is make a request, if its only `5ms` then it means there is no account with that email. if its `100ms` then it means the account exists but the password is incorrect. and thus they got information about your user count